CHANGELOG.md

```diff
@@ -1,3 +1,190 @@
+## Release 0.14.0
+
+Highlights:
+
+This release comes with two main new features:
```

Comment:
Weird to have Highlights: followed immediately by this text. Suggest dropping Highlights?